nservicebus/upgrades/azure-functions-service-bus-in-process-isolated-worker.md

```diff
@@ -0,0 +1,163 @@
+---
+title: Migrating Azure Functions (in-process) to (Isolated Worker)
+summary: Instructions on how to migrate Azure Functions (in-process) to Azure Functions (Isolated Worker)
+component: ASBFunctionsWorker
+reviewed: 2024-07-29
+related:
+ - nservicebus/hosting/azure-functions-service-bus/in-process
+ - nservicebus/hosting/azure-functions-service-bus
+isUpgradeGuide: true
+---
+
+## Overview
+
+Microsoft has confirmed that .NET 8.0 will be the last LTS to support the in-process hosting model. It is highly recommended to migrate to the isolated worker hosting model.  This guide provides steps to help you migrate from in-process to isolated worker.
+
+## Key differences
+
+- **In-process**: Functions run within the Azure Functions runtime, sharing the same process and AppDomain.
+- **Isolated worker**: Functions are isolated from the Azure Functions runtime and run in a separate process.
+
+Microsoft provides more detailed information about the [difference between the in-process and isolated worker model](https://learn.microsoft.com/en-us/azure/azure-functions/dotnet-isolated-in-process-differences).
+
+This migration guide is based on the `NServicebus.AzureFunctions.InProcess.ServieBus 2.x` component [example](/samples/azure-functions/service-bus/?version=asbfunctions_2) and the `NServiceBus.AzureFunctions.Worker.ServiceBus 3.x` component [example](/samples/azure-functions/service-bus-worker/?version=asbfunctionsworker_3) as reference.
+
+## Update project file
+
+### Update framework
+
+  Change the `TargetFramework` to `net6.0` or higher in your `.csproj` file
+
+```xml
+<PropertyGroup>
+  <TargetFramework>net6.0</TargetFramework>
+</PropertyGroup>
+```
+
+### Remove package references
+
+Remove the following package references from your `.csproj` file.
+
+- `Microsoft.NET.Sdk.Functions`
+- `Micrsosoft.Azure.Functions.Exension`
+- `Microsoft.Azure.Webjobs`
+- `Microsoft.Azure.Webjobs.extensions.http`
+
+### Add package references
+
+Add the following package references to your `.csproj` file.
+
+```xml
+<ItemGroup>
+  <PackageReference Include="Microsoft.Azure.Functions.Worker" Version="1.*" />
+  <PackageReference Include="Microsoft.Azure.Functions.Worker.Sdk" Version="1.*" />
+  <PackageReference Include="Microsoft.Azure.Functions.Worker.Extensions.Http" Version="3.*" />
+  <PackageReference Include="NServiceBus.AzureFunctions.Worker.ServiceBus" Version="3.*" />
+</ItemGroup>
+```
```

Comment:
Since we are using the isolated worker process sample as a reference, would you also like to include that the `<OutputType>Exe</OutputType>` be added in the csproj file?